images/win/scripts/Installers/Install-SQLExpress.ps1

```diff
@@ -0,0 +1,54 @@
+#Download web-installer
+Set-Item .\MaxMemoryPerShellMB 4096
+$installerUrl = "https://go.microsoft.com/fwlink/?linkid=866658"
+$installerName = "SQL2019-SSEI-Expr.exe"
+$installerPath = "${env:Temp}\$installerName"
+Write-Host "Download web-installer"
+(New-Object System.Net.WebClient).DownloadFile($installerUrl, $installerPath)
```

Comment:
I think we should rewrite `Install-VsixExtension`  in more general way and use retry logic here.

Reply:
Done. Please review